kitten-scientists.js

```diff
@@ -84,10 +84,12 @@ var options = {
                 megalith: {require: false, stock: 0, type: 'craft', enabled: false}
             }
         },
-        // @TODO: enable other races for trading
         trade: {
-            enabled: true, trigger: 0.90, items: {
-                zebras: {trigger: 0.95, max: 'titanium', require: false, season: 'summer', enabled: true}
+            enabled: true, trigger: 0.95, items: {
+                griffins: {trigger: 0.99, require: 'wood', summer: true, autumn: false, winter: false, spring: false, enabled: false},
+                zebras: {trigger: 0.99, require: false, summer: true, autumn: false, winter: false, spring: false, enabled: true},
+                dragons: {trigger: 0.99, require: 'titanium', summer: true, autumn: true, winter: false, spring: true, enabled: false},
+
```

Comment:
Yea. I am not sure how that one change got in.. I will make sure to revise
this series and fix up the complaints.
On May 2, 2015 12:31 PM, "Cameron Condry" <notifications@github.com> wrote:

> In kitten-scientists.js
> <https://github.com/cameroncondry/cbc-kitten-scientists/pull/34#discussion_r29550054>
> :
>
> >          trade: {
> > -            enabled: true, trigger: 0.90, items: {
> > -                zebras: {trigger: 0.95, max: 'titanium', require: false, season: 'summer', enabled: true}
> > +            enabled: true, trigger: 0.95, items: {
> > +                griffins: {trigger: 0.99, require: 'wood', summer: true, autumn: false, winter: false, spring: false, enabled: false},
> > +                zebras: {trigger: 0.99, require: false, summer: true, autumn: false, winter: false, spring: false, enabled: true},
> > +                dragons: {trigger: 0.99, require: 'titanium', summer: true, autumn: true, winter: false, spring: true, enabled: false},
> > +
>
> I added all of the trade options in the latest version, and would like to
> at least see the same options in a disabled state existing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cameroncondry/cbc-kitten-scientists/pull/34/files#r29550054>
> .
>